ocaml/typing/persistent_env.ml

```diff
@@ -97,14 +98,19 @@ type global_name_info = {
   gn_mentioned_by : Global_module.Name.t; (* For error reporting *)
 }
 
+type import_kind =
+  | Normal of {
+    imp_params : Global_module.Parameter.t list;
+    imp_arg_for : Global_module.Parameter.t option;
+    imp_impl : CU.t;
+    imp_raw_sign : Signature_with_global_bindings.t;
```

Comment:
Discussed offline - we think it's simpler to lift this out to be under `import` directly.